src/model/rpc.js

```diff
@@ -75,13 +75,13 @@ module.exports = class BtpRpc extends EventEmitter {
         contentType: btpPacket.MIME_APPLICATION_OCTET_STREAM,
         data: Buffer.from([])
       }, {
-        protocolName: 'auth_token',
+        protocolName: 'auth_username',
         contentType: btpPacket.MIME_TEXT_PLAIN_UTF8,
-        data: Buffer.from(authToken, 'utf8')
+        data: Buffer.from(authUsername, 'utf8')
```

Comment:
should default to `''`, because the username may not be specified

Reply:
if it's not specified in the URL then `parsedBtpUri.username` would be `''` so it's safe, but I'll add unit tests for both empty authUsername and empty authToken.